browser/html/cool-help.html

```diff
@@ -6,573 +6,777 @@
     .productname { font-weight:bold; font-style:italic; }
     .ui { font-weight:bold; }
     .def { font-weight:bold; }
-    .toc-h2 { font-weight:bold; }
-    .toc-h3 { margin-left:0.1965in; }
+    /* Style for toc-h2 and toc-h3 */
+    .toc-h2 button,
+    .toc-h3 button {
+        border-radius: 5px; /* Add rounded corners */
+        transition: background-color 0.3s ease; /* Smooth transition effect */
+        font-family: Arial, sans-serif; /* Use a common font family */
+        font-size: 16px; /* Font size */
+        color: var(--color-main-text);
+    }
+
+    /* Hover effect for toc-h2 and toc-h3 button */
+    .toc-h2 button:hover,
+    .toc-h3 button:hover {
+        background-color: #e0e0e0; /* Darker gray background on hover */
+    }
+
+    /* Style for toc-h2 and toc-h3 button text when clicked */
+    .toc-h2 button:active,
+    .toc-h3 button:active {
+        color: #555; /* Darker text color when clicked */
+    }
+
+    /* Style for toc-h2 and toc-h3 button text when focused */
+    .toc-h2 button:focus,
+    .toc-h3 button:focus {
+        outline: none; /* Remove outline when focused */
+    }
+
+    /* Add margin to the paragraph to separate each item */
+    .toc-h2,
+    .toc-h3 {
+        margin-bottom: 2px; /* Adjust margin bottom */
+    }
+
     .screenshot { text-align:center; margin: 0.1in; }
     .screenshot > img { max-width: 100%; height: auto; }
     .blue { color:#63bbee; }
     button.border-0 { font-family: "Helvetica Neue", sans-serif; background: none!important; border: none; padding: 0!important; color: -webkit-link; cursor: pointer; text-decoration: underline; font-size: 1em; line-height: 1.5em; }
     button:active { color: -webkit-activelink; }
     button:focus { outline: -webkit-focus-ring-color auto 1px; }
+
+    #online-help-search-input {
+      width: 30%;
+      padding: 5px 10px 5px 15px;
+      border: 1px solid;
+      border-radius: 5px;
+      font-size: 16px;
+      outline: none;
+    }
+    #online-help-search-input:focus {
+      box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
+    }
+    #online-help-search-input::placeholder {
+      color: #999;
+    }
     [data-theme='dark'] span.cool-annotation-menu { filter: invert(1);}
 </style>
+
+<div class="search-container">
+  <input size="20" id="online-help-search-input" placeholder="Search..." />
```

Comment:
placeholder="Search..." is not localisable.

Reply:
@timar i did not understand your comment.

I guess you are trying to suggest that we should not use `...` in placeholder for any translation errors right ?

Reply:
See https://github.com/CollaboraOnline/online/pull/8360
We'll need that commit in co-23.05, too.